src/internal/SetupResumeAcceptor.cpp

```diff
@@ -143,8 +143,19 @@ void SetupResumeAcceptor::processFrame(
         break;
       }
 
-      removeConnection(transport);
-      onResume(std::move(transport), std::move(resumeParams));
+      if (!onResume(transport, std::move(resumeParams))) {
+        transport->outputFrameOrEnqueue(frameSerializer->serializeOut(
+            Frame_ERROR::connectionError("Resumption Failure")));
```

Comment:
`onResume` could return false for a number of reasons.  It just happens that in the current version, resumption is not supported.  Hence I went with a generic "Resumption Failure" message.  Later, we could augment this code to get the actual failure reason from `onResume`